pkg/cmd/controller/root.go

```diff
@@ -18,8 +18,15 @@ func NewCommand() *cobra.Command {
 		Short: "OCI image build controller using buildkit",
 	}
 	cmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", "hephaestus.yaml", "configuration file")
+
+	runGCCmd := newRunGCCommand()
+	runGCCmd.Flags().Bool("enabled", true, "Enable the auto image builder clean up.")
```

Comment:
Do we need an enabled flag in the CLI? Can we just enable/disable the cronjob?

Reply:
yea I was thinking that was kinda redundant